packages/graph/src/Vertex.ts

```diff
@@ -23,6 +23,7 @@ export class Vertex extends SVGWidget {
         this._icon = new Icon();
         this._textBox = new TextBox();
         this._annotationWidgets = {};
+        this.pos({ x: undefined, y: undefined });
```

Comment:
Why is this necessary?

Reply:
As it defaults to {x: 0, y: 0} which may be a legitimate initial position...

Reply:
@jbrundage bump